### PR TITLE
Add Aceternity UI components

### PR DESCRIPTION
**Pull request by JakeJeongAiden**
Add components from https://ui.aceternity.com/components.

### CODE REVIEW

File: apps/chameleon/app/testground/page.tsx
Comment (Balance8) [inline apps/chameleon/app/testground/page.tsx:1]: Great work testing, lets not commit this code.

**Review by Balance8 — CHANGES_REQUESTED** [review]

File: apps/chameleon/app/testground/page.tsx
Comment (JakeJeongAiden) [reply apps/chameleon/app/testground/page.tsx:1]: Thanks! 
I deleted the page.tsx from testground

**Review by Balance8 — APPROVED** [review]
Nice work!

**Review by BBruington — APPROVED** [review]
Bigger Pog

**Review by DinDin985 — APPROVED** [review]
Lgtm! 👍

**Pull request merged**